.github/workflows/ci.yml

```diff
@@ -0,0 +1,130 @@
+name: CI
+on: [push]
+
+jobs:
+  lint:
+    name: Linters
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          cache: 'yarn'
+          node-version-file: '.node-version'
+
+      - name: Install Yarn packages
+        run: yarn install --check-files --pure-lockfile
+
+      - name: Lint files
+        run: bin/lint
+
+  scan:
+    name: Security Analysis
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Scan code vulnerabilities
+        run: bin/scan
+
+  test:
+    name: Test
+    needs:
+      - lint
+      - scan
+    runs-on: ubuntu-latest
+    env:
+      DATABASE_URL: postgres://postgres:example@localhost:5432/app_test
```

Comment:
Those env's needs to come from env's set on github account?  Do you think worth have a `.md` explaing that? If it's the case. Or we don't need that?

Reply:
The environments defined in the CI/CD configuration file do not need to come from the environments defined in the GitHub account. They are specific to running the CI/CD pipeline and can be configured directly in the configuration file.

I think the environment variables in the CI/CD configuration file are used to configure the execution environment for the pipeline steps, such as setting the database connection credentials for the tests.